bfex/components/scrapper/scrapper.py

```diff
@@ -0,0 +1,102 @@
+from bs4 import BeautifulSoup
+import requests
+import json
+import re
+
+base_url = "https://www.ualberta.ca/science/about-us/contact-us/faculty-directory/"
+
+ORCID = 0
+RESEARCHID = 1
+PROFILE = 2
+
+class Scrapp(object):
+    def __init__(self):
+        self.title = ""
+        self.text = ""
+        self.date = ""
+        self.meta_data = {}
+
+    def set_title(self, title):
```

Comment:
in regards to our slack convo. A scrapp should have a `self.formatted_name` or something, you don't have to code how we get it now but we have all this info about a prof and not their name attached to it.
we should also have a `self.data_source`